### ISSUE TITLE
better ExtractAndCombData warning message

### ISSUE THREAD

**Issue by goldingn** [zoonproject/zoon#384]
This pops up a lot:

```
Warning message:
In ExtractAndCombData(x, covariate.output[[1]]) :
  38 extracted covariate values are NA. This may cause issues for some models
```

but it isn't immediately obvious that it refers to *occurrences* falling where there are missing covariate values. Could do with rewording.

**Comment by AugustT** [comment]
Yeah, it makes sense to me but maybe not to a new user. Maybe change to something like "X occurrence points fall outside your covariate raster and therefore have NA values for covariates..."

**Comment by goldingn** [comment]
yeah, that's perfect
